lib/file_transfer_agent/file_transfer_agent.js

```diff
@@ -379,24 +379,20 @@ function file_transfer_agent(context)
     {
       // Remove all files inside tmp folder
       var matchingFileNames = glob.sync(path.join(meta['tmpDir'], meta['srcFileName'] + "*"));
-      for (var matchingFileName of matchingFileNames)
-      {
-        fs.unlinkSync(matchingFileName);
+      for (var matchingFileName of matchingFileNames) {
+          fs.unlinkSync(matchingFileName);
```

Comment:
This unlink is not working, at least not on Linux.  Not sure why as it's no different from above.  However, in this PR the following rmdir fails because the file still exists:

/code/node_modules/snowflake-sdk/lib/file_transfer_agent/file_transfer_agent.js:388
              throw (err);
              ^

[Error: ENOTEMPTY: directory not empty, rmdir '/tmp/tmpcTyqEs'] {
  errno: -39,
  code: 'ENOTEMPTY',
  syscall: 'rmdir',
  path: '/tmp/tmpcTyqEs'
}
root@e013314ebd87:/code#

Reply:
the `ENOTEMPTY` issue is not localized to Linux, also happens on Mac. This PR breaks the driver.